src/lib/utils/synapseTypes/File/UploadDestination.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * The upload destination contains information to start an upload of a file generated according to the underlying StorageLocationSetting.
+ *
+ * https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/file/UploadDestination.html
+ */
+export interface UploadDestination {
```

Comment:
UploadDestination types will be used in the new title bar to denote the storage location of a container entity